manim/mobject/types/vectorized_mobject.py

```diff
@@ -2216,11 +2261,13 @@ def construct(self):
                     )
                     middle_row.add(circ)
 
-                sq = DashedVMobject(Square(1.5, color=RED))
-                penta = DashedVMobject(RegularPolygon(5, color=BLUE))
-                bottom_row = VGroup(sq, penta)
+                func1 = FunctionGraph(lambda t: t**5,[-1,1],color=WHITE)
+                func_even = DashedVMobject(func1,num_dashes=6,even_lengths=True)
+                func_stretched = DashedVMobject(func1, num_dashes=6, even_lengths=False)
```

Comment:
`even_lengths` -> `equal_lengths`

```suggestion
                func_even = DashedVMobject(func1, num_dashes=6, equal_lengths=True)
                func_stretched = DashedVMobject(func1, num_dashes=6, equal_lengths=False)
```

Reply:
fixed in the next commit